consumer/handler/func_test.go

```diff
@@ -0,0 +1,24 @@
+package handler_test
+
+import (
+	"testing"
+
+	"github.com/heetch/felice/consumer/handler"
+	"github.com/heetch/felice/message"
+	"github.com/stretchr/testify/require"
+)
+
+// HandleMessageFn can be used as a Handler
+func TestHandleMessageFn(t *testing.T) {
+	calls := make([]bool, 0)
```

Comment:
📖 Just in case, `var calls []bool` works (almost) the same way, `append` accepts a zero value.

Reply:
Ah.. now that I didn't know.  Thanks.